static/app/views/settings/organizationMembers/inviteBanner.tsx

```diff
@@ -68,6 +80,15 @@ export function InviteBanner({
     });
   }, [api, organization, promptsFeature, hideBanner]);
 
+  useEffect(() => {
+    const params = new URLSearchParams(location.search);
```

Comment:
i do think that using the `query-string` package here is more inline with the rest of the frontend codebase, and would just end up looking something like

```
const inviteMissingMembers = qs.parse(location.search).inviteMissingMembers
```

but I asked in #discuss-frontend if there is a super hard preference.

Reply:
i can use qs